src/proteus/atmos_clim/janus.py

```diff
@@ -199,7 +199,7 @@ def RunJANUS(atm, dirs:dict, OPTIONS:dict, hf_row:dict, hf_all:pd.DataFrame,
 
         # run JANUS
         atm = MCPA_CBL(dirs, atm, trppD, rscatter, method=search_method, atol=tol,
-                        atm_bc=int(OPTIONS["F_atm_bc"]), T_surf_guess=float(T_surf_old)-0.5, T_surf_max=float(T_surf_max))
+                        atm_bc=int(OPTIONS["F_atm_bc"]), T_surf_guess=float(T_surf_old)-0.1, T_surf_max=float(T_surf_max))
```

Comment:
I am a bit afraid about this change. How does this affect all the results stored in the example folder? Is it why the earthdemo helpfile is changin?

Reply:
This is intended to improve performance, because often the next temperature solution is only ~0.1 K cooler than the previous one. Therefore, by moving the initial guess closer to the previous solution the model should have to do fewer steps in order to determine the next temperature. In theory it shouldn't change the model output since the solver should be finding the same solution regardless of the initial guess.

However, we can revert this and leave it for a later work if you feel that that's best.

Reply:
Makes sense. I think you can leave it like this.